tree_test.go

```diff
@@ -1168,3 +1168,62 @@ func TestRustBanderwagonBlock48(t *testing.T) {
 		t.Fatal("deserialized proof didn't verify")
 	}
 }
+
+func BenchmarkEmptyHashCodeCachedPoint(b *testing.B) {
+	_ = GetConfig()
+	const codeHashVectorPosition = 3 // Defined by the spec.
+
+	testCases := []struct {
+		name     string
+		hashCode string
+	}{
+		{name: "emptyHash", hashCode: "c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"},
+		{name: "nonEmptyHash", hashCode: "4242420186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"},
```

Comment:
We compare the empty code hash against a slightly modified hash that wouldn't leverage the cached point, which allows seeing in the output the performance difference between the optimized and unoptimized case.